pkg/github/commenter/ghcommenter.go

```diff
@@ -213,25 +213,83 @@ func (ghc *GitHubCommenter) UpdatePendingCommentRecords(org, repo string, number
 	}
 }
 
-func (ghc *GitHubCommenter) ClearPendingRecord(org, repo string, number int, sha string, record *models.PullRequestComment) {
+func (ghc *GitHubCommenter) ValidatePendingRecordComment(org, repo string, number int, sha string, commentType models.CommentType) (bool, error) {
+	// check for this record in our table
```

Comment:
can you pls put the comment above the `func` to say "ValidatePendingRecordComment checks for this record in our table, ..."

Reply:
and please include what the bool return value means.  I think false means didn't do anything (either due to error or not enough time elapsed) and true means it updated the last comment.

Reply:
Just a humble :-) opinion on the name `ValidatePendingRecordComment`, ... are you validating or updating the last attempt?

Reply:
If the pending comment is valid the function will update the last attempt and return true.  If the pending comment is not valid it will not make any update and return false.  I can move the doc for that and change the name to ValidateAndUpdatePendingRecordComment if that is clearer.